recipes/license_server.rb

```diff
@@ -66,6 +72,11 @@
 b = ruby_block 'license cleanup' do
   block do
     license_groups.each do |type, licenses|
+      existing_directory = Dir.glob("#{node['splunk']['home']}/etc/licenses/*")
```

Comment:
I know that we only support one type now, but this logically should be outside of the license_groups.each loop